analysis/metrics.py

```diff
@@ -49,7 +49,7 @@ class SumMetrics:
 
 
 @dataclass
-class PerPartitionUtility:
+class PerPartitionUtilityMetrics:
```

Comment:
Optional: Do we need the `Utility` in the name?

It seems it would be more consistent to remove it

Reply:
Agree, removed